go.mod

```diff
@@ -5,13 +5,14 @@ go 1.13
 require (
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
 	github.com/apache/dubbo-go v0.1.2-0.20200224151332-dd1a3c24d656
+	github.com/coreos/etcd v3.3.13+incompatible
+	github.com/go-errors/errors v1.0.1
```

Comment:
Why do we add this package?
"github.com/pkg/errors" might be enough?

Reply:
yes, it' mistake, i will remove it.